shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.js

```diff
@@ -54,7 +57,7 @@ const completeWorkItem = async ({
 
   await applicationContext.getPersistenceGateway().putWorkItemInUsersOutbox({
     applicationContext,
-    section: user.section,
+    section: user.section ? user.section : DOCKET_SECTION,
```

Comment:
I'm curious about the real-life use case that required this change. Was this required for tests? I'd think any court user with access to serve court-issued documents would have a section.

Reply:
This fixes a bug when pdf is uploaded and save entry without serving and then later served by clicking on the link within docket record.  Without the ternary, when you serve it from the docket record, it shows up under the docket clerk's Document QC Processed tab, but does not show up under the Section Document QC Processed tab.  This will fix that issue.

Reply:
fixed petitionsclerk issue and no longer need this line, reverted back to original line of just using user.section only